weather.noaa/resources/.gitignore

```diff
@@ -0,0 +1 @@
+/settings.bak.xml
```

Comment:
Sorry this file has to be removed from the submission

Reply:
This should be removed now.